COMMERCE-10604 - Cannot PATCH a sku's ERC via API

victorcdp opened the pull request:
Related Issue: [COMMERCE-10604](https://issues.liferay.com/browse/COMMERCE-10604)

The ERC was being updated with the saved SKU ERC instead of the one sent by the API.
Proposed fix is to change it so the ERC is updated by the one sent by the API

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

matthewchan1 commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest(master)/4471/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-21.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4471/jenkins-report.html">here</a>.</b></p>

brianikim commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-16.liferay.com/job/test-portal-source-format/1736/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-16.liferay.com//userContent/jobs/test-portal-source-format/builds/1736/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 1 minute 48 seconds 222 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/178df0ee86f65868f2597c34e103a6e2bb9a3ba4">178df0ee86f65868f2597c34e103a6e2bb9a3ba4</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/matthewchan1/liferay-portal/tree/LPS-99216">LPS-99216</a><br/>Branch GIT ID: <a href="https://github.com/matthewchan1/liferay-portal/commit/85538249a416faf5021e4751e224e3e53be7f715">85538249a416faf5021e4751e224e3e53be7f715</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-16.liferay.com/job/test-portal-source-format/1736/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-16.liferay.com/userContent/jobs/test-portal-source-format/builds/1736/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 24 out of 26 jobs passed in 1 hour 6 minutes 12 seconds 341 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d44939351a741a7829f8ea939bf6829d6383c153">d44939351a741a7829f8ea939bf6829d6383c153</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/cfda345def224774cde1fc5ec7948658ab7c4c58">cfda345def224774cde1fc5ec7948658ab7c4c58</a></p>24 out of 26 jobs PASSED<div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest(master)/4471/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305092/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/7</strong></strike></a></li></ul></div><details><summary><strong>24 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305088/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/271020/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305094/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/309979/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/307959/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/181216/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/307958/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/309978/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305093/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/181214/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305090/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305089/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/181211/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/181213/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/181210/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/181208/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/306346/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68388/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/181209/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305091/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/307957/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305095/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/181212/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/181215/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-21.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4471/jenkins-report.html">here</a>.</h5><hr/><h4>This pull contains no unique failures.</h4><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">acceptance upstream results</a> at d44939351a741a7829f8ea939bf6829d6383c153:</strong></summary><ol><li><div><h5><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305092/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/7</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/305092//consoleText">AXIS_VARIABLE=0,label_exp=!master #305092</a><ol><li><div><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/305092//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_FormRules_ConfigureAutofillRule_">LocalFile.FormRules#ConfigureAutofillRule</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1564785248984/test-portal-acceptance-pullrequest(master)/4471/functional-tomcat90-mysql57-jdk8/7/0/LocalFile.FormRules_ConfigureAutofillRule/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1564785248984/test-portal-acceptance-pullrequest(master)/4471/functional-tomcat90-mysql57-jdk8/7/0/LocalFile.FormRules_ConfigureAutofillRule/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1564785248984/test-portal-acceptance-pullrequest(master)/4471/functional-tomcat90-mysql57-jdk8/7/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li></ol></details></details></html>

brianikim commented:
https://github.com/SamZiemer/liferay-portal/pull/627

georgel-pop-lr committed:
LPS-167763 Display correct results when permission scope is changed in a role

JanisZhang committed:
LPS-171453 Remove optional<> usages in InfoField

JanisZhang committed:
LPS-171453 Remove optional<> usages in CollectionQuery

JanisZhang committed:
LPS-171453 Remove optional<> usages in InfoItemIdentifier

JanisZhang committed:
LPS-171453 Apply usages

JanisZhang committed:
LPS-171453 Remove optional<> usages in InfoItemFormVariation

Commit:
LPS-166921 Remove automatically-set aria-labelledby from menu when triggered by an input-localized taglib

Commit:
LPS-166921 Add internationalized aria labels so screen readers announce translations statuses

JanisZhang committed:
LPS-171453 Remove Optional<> in InfoListRendererContext

JanisZhang committed:
LPS-171453 Apply usages

carolmariaabb committed:
LPS-171054 Expose the checkModelResourcePermission method

carolmariaabb committed:
LPS-171054 Check permission when executing the standalone object action

carolmariaabb committed:
LPS-171053 Check permission when listing the object entry

danvb112 committed:
LPS-171100 Move FDSProps transformer components to a new directory

danvb112 committed:
LPS-171100 Extract NotificationQueueEntryToDataRenderer into your own component

carolmariaabb committed:
LPS-171052 Populate object definition model resource actions including standalone object actions

carolmariaabb committed:
LPS-171052 Do not allow standalone actions for system objects

BarbaraCabrera committed:
LPS-171382 Add the Remove Thumbnail action for the Utility Pages

BarbaraCabrera committed:
LPS-171382 SF create a new variable to avoid duplicated code

Commit:
LPS-166921 Use CSS class to prevent aria attributes of list items from being removed

Commit:
LPS-166921 Add triggerAriaLabel attribute to liferay-ui:icon-menu taglib

lfbesada committed:
LPS-171382 Don't throw an exception.

IMHO it is a good idea to cache the update permission check, but we shouldn't throw an exception in any case, just do not allow the affected actions. So let's cache it in a variable and capture the exception if it happens.

lfbesada committed:
LPS-171382 Use the bean reference instead of the Util

ces-sontruong committed:
LPS-171297 Check Custom User Attribute with locale

ruben-pulido committed:
LPS-169503 Set id as 'addLayoutDialog' so modal closes after performing copy page action

Co-authored-by: Victor Galan <victor.galan@liferay.com>

ruben-pulido committed:
LPS-169503 Remove namespace from id when firing closeModal event

Co-authored-by: Victor Galan <victor.galan@liferay.com>

ruben-pulido committed:
LPS-169503 Adapt usages

Co-authored-by: Victor Galan <victor.galan@liferay.com>

ruben-pulido committed:
LPS-169503 Include url to set permissions

ruben-pulido committed:
LPS-169503 build.gradle

veroglez committed:
LPS-169503 Add new dropdown options

veroglez committed:
LPS-169504 Open modal when the delete option is selected

ruben-pulido committed:
LPS-169503 Include url to copy layout

ruben-pulido committed:
LPS-169503 For each layout, determine, if layout is deleted, which layout should be selected next, and pass it to LayoutActionProvider when getting the actions.

The rules for determining the next layout are:
- If the layout has siblings, the previous sibling is selected, if available.
- Else, if the layout has a parent layout, the parent layout is selected.
- If the layout has siblings, but the previous layout is not available, (the layout is the first node in the tree), select the second layout if available.

Example:

layout -> afterDeleteSelectedLayout

/p1 -> /p2
/p2 -> /p1
/p3 -> /p2
/p4 -> /p3
/p1_1 -> /p1
/p2_1_1 -> /p2_1
/p2_1_2 -> /p2_1_1
/p2_1 -> /p2
/p2_2 -> /p2_1
/p2_3 -> /p2_2
/p2_4 -> /p2_3

ruben-pulido committed:
LPS-169503 Include URL for deleting a page

ruben-pulido committed:
LPS-169503 Perform a fetch when deleting a page and navigate to the redirect location.

Co-authored-by: Victor Galan <victor.galan@liferay.com>

ruben-pulido committed:
LPS-169503 Set redirect for deleteteLayoutURL.

If the current page being viewed or edited corresponds to the page being deleted, redirect to afterDeleteSelectedLayout if available or to "Site Builder > Pages" if not available.

Otherwise, redirect to the current location.

ruben-pulido committed:
LPS-169503 Prevent NoSuchLayoutException when selected layout no longer exists

Co-authored-by: Lourdes Fernandez Besada <lourdes.fernandez@gmail.com>

felipelorenz committed:
LPS-162052 LPS-165714 portal-search-web: Modified - Extend BucketDisplayContext and delete overriding fields

https://issues.liferay.com/browse/LPS-165714

lfbesada committed:
LPS-171382 Let's update first the LayoutUtilityPageEntry in order to not lose the FileEntry if an error occurs during updating

lfbesada committed:
LPS-171382 Prevent from updating when there isn't preview file

lfbesada committed:
LPS-171382 We don't need to create a folder anymore, the thumbnails will be creating directly in the repository folder

sandrodw3 committed:
LPS-169503 SF

sandrodw3 committed:
LPS-169503 Make copy modal smaller

DiegoHu97 committed:
LPS-171684 Add WidgetsTree component

sandrodw3 committed:
LPS-171684 Render component when feature flag is enabled

lfbesada committed:
LPS-171382 Add a new method signature to allow set the name to the copied fileEntry and deprecate the old method signature

lfbesada committed:
LPS-171382 Remove deprecated method

lfbesada committed:
LPS-171382 BuildService

lfbesada committed:
LPS-171382 We don't need to delete the folder anymore

lfbesada committed:
LPS-171382 Adapt usages

lfbesada committed:
LPS-171382 Change the way copying the preview FileEntry in order to do get same that when we set it using the UI

felipelorenz committed:
LPS-162052 LPS-165714 portal-search-web: Modified - Replace field with BucketText

https://issues.liferay.com/browse/LPS-165714

Commit:
LPS-166921 Try first to get the language name with the full language key

Commit:
LPS-166921 Use hasClass function to check if menu was triggered by input-localized taglib

Commit:
LPS-166921 Add language keys

Commit:
LPS-166921 We don't need to announce arrow navigation instructions because they are already announced when menu opens

felipelorenz committed:
LPS-162052 LPS-165714 portal-search-web: Modified - Replace field with filterValue

https://issues.liferay.com/browse/LPS-165714

Commit:
LPS-166921 Build languages

ruben-pulido committed:
LPS-169503 After copying layout from page tree, redirect to the previous location, instead of always redirecting to pages navigation

lfbesada committed:
LPS-171382 Auto

Commit:
LPS-167730 Convert method reference to ServiceTrackerMap

JanisZhang committed:
LPS-171453 Apply usages

Commit:
LPS-167674 Fix filter if the value has an apostrophe

Hanlf committed:
LPS-169410 Move some headless tests to Lima

Hanlf committed:
LPS-169410 Add headless tests into lima batches

Hanlf committed:
LPS-169410 Update relevant query to contains lima headless tests

ealonso committed:
LPS-171297 SF

JoyceWang08211 committed:
LRQA-78789 Add target to import custom database dump

ruben-pulido committed:
LPS-171720 Include redirect as parameter when calling the load more pages endpoint

lfbesada committed:
LPS-171145 Set the related assetCategoryId in the AssetEntryQuery instead of in the SearchContext

DiegoHu97 committed:
LPS-171684 Create displayContext

lfbesada committed:
LPS-171145 SF as used

lfbesada committed:
LPS-171145 When assetCategoryRule is set as anyAssetCategoryOfTheSameVocabulary we should add as anyCategoryIds the others categories of the same vocabulary and let the context's category as allCategoryIds

lfbesada committed:
LPS-171145 When assetCategoryRule is set as specificAssetCategory and the specificAssetCategoryId doesn't match with the context's assetCategory we should set as allCategoryIds the both categories

lfbesada committed:
LPS-171145 Apply the assetCategoryRule only if the feature flag is enabled

ealonso committed:
LPS-171684 Removes feature flag

ealonso committed:
LPS-171684 Removes no needed PortletLister classes

felipelorenz committed:
LPS-162052 LPS-165714 portal-search-web: Modified - delete range because it is similar to bucketText

https://issues.liferay.com/browse/LPS-165714

felipelorenz committed:
LPS-162052 LPS-165714 portal-search-web: Modified - delete the assert because it is not useful

https://issues.liferay.com/browse/LPS-165714

pedro-oliveira446 committed:
LPS-171226 adding 'backURL' parameter in tab url

danielCamara02 committed:
LPS-171556 Add CanOrderTypeFacetTerms test

tinatian committed:
LPS-167730 SF, simplify

tinatian committed:
LPS-167730 SF, simplify

tinatian committed:
LPS-167730 SF, simplify

ealonso committed:
LPS-171874 Adds layoutSelectorURL

ealonso committed:
LPS-171874 buildTaglibs

ealonso committed:
LPS-171874 build.gradle

ealonso committed:
LPS-171874 Adds layoutSelectorURL to ddm_form.js context

ealonso committed:
LPS-171874 Auto SF

ealonso committed:
LPS-171874 Uses layoutSelector to open an item selector instead of creating a custom dialog to select layouts

ealonso committed:
LPS-171874 baseline

JessieZhao0714 committed:
LPS-170504 Look up clientId via externalReferenceCode

sandrodw3 committed:
LPS-171684 We don't have root item anymore

sandrodw3 committed:
LPS-171684 No needed

sandrodw3 committed:
LPS-171684 Add proptypes

sandrodw3 committed:
LPS-171684 SF

Commit:
LPS-171908 Remove property ignore on ObjectFields related the Aggregation Fields

felipelorenz committed:
LPS-162052 LPS-165715 portal-search-web: Modified - Replace ModifiedFacetTermDisplayContext with BucketDisplayContext

https://issues.liferay.com/browse/LPS-165715

felipelorenz committed:
LPS-162052 LPS-165715 portal-search-web: Modified - Delete ModifiedFacetTermDisplayContext

https://issues.liferay.com/browse/LPS-165715

Commit:
LPS-166110 batch-engine - add update strategy

Commit:
LPS-166110 batch-planner - relly to update strategy

Commit:
LPS-166110 batch-planner - JS/JSP - show dropdown with update strategy

Commit:
LPS-166110 portal-language - add update strategy language keys

Commit:
LPS-166110 portal-language - Autogenerated - buildLang

danielCamara02 committed:
LPS-171556 SF

felipelorenz committed:
LPS-162052 LPS-165714 portal-search-web: Modified - SF

https://issues.liferay.com/browse/LPS-165714

felipelorenz committed:
LPS-162052 LPS-165715 portal-search-web: Modified - Change template class name

https://issues.liferay.com/browse/LPS-165715

JanisZhang committed:
LPS-171453 Apply usages

lfbesada committed:
LPS-171959 Send to the front side all the existing form type options setting the information about the user permissions in a property

lfbesada committed:
LPS-171959 Preserve the old behavior when feature flag is disabled

lfbesada committed:
LPS-171959 Auto

lfbesada committed:
LPS-171960 Add language key

lfbesada committed:
LPS-171960 BuildLang

JoyceWang08211 committed:
LRQA-78789 Add new test suite

JanisZhang committed:
LPS-172109 Remove Optional usages in GetCollectionFieldMVCResourceCommand

lfbesada committed:
LPS-171960 Adds a method to check if the user has the ADD permission

lfbesada committed:
LPS-171960 Implements the new hasAddPermission method, the check permissions logic is copied from ObjectEntryServiceImpl.addObjectEntry

lfbesada committed:
LPS-171960 When is VIEW mode and the user hasn't the ADD permission, display a message instead of the form content

lfbesada committed:
LPS-171960 When isn't VIEW mode and the user hasn't the VIEW permission, display a message instead of the form content

lfbesada committed:
LPS-171960 SF, remove never thrown exception, we are asking if contains permission not checking

lfbesada committed:
LPS-171960 Adapt usages

Commit:
LPS-171684 Modify locators to adapt new markup

lfbesada committed:
LPS-171960 Hide the new behavior under the feature flag

lfbesada committed:
LPS-171960 Baseline

Commit:
LPS-171684 Modify macro to adapt new behavior

lfbesada committed:
LPS-171960 SF rename

JanisZhang committed:
LPS-172109 Remove Optional usages in FileEntryMetadataOpenGraphTagsProvider

lfbesada committed:
LPS-171960 We must be restrictive and return false if there is not themeDisplay

Commit:
LPS-166921 Fix path to fix a11y issues

julschong committed:
LPS-171930 Remove Stream usages in GraphQLServletExtender

JanisZhang committed:
LPS-172109 Remove Optional usages in SegmentsExperienceUtil

JanisZhang committed:
LPS-172109 Remove Optional usages in OpenGraphSettingsDisplayContext

JanisZhang committed:
LPS-172109 Remove Optional usages in ContentPageLayoutEditorDisplayContext

julschong committed:
LPS-171930 Remove Stream usages in VulcanPropertyFilter

julschong committed:
LPS-171930 Remove Stream usages in FieldsQueryParamContextProvider

crodriguezy committed:
LPS-172057 Parse templatenodes numeric data just like its done in NumericDDMFormFieldUtil

julschong committed:
LPS-171930 Remove Stream usages in JsonMappingExceptionMapper

julschong committed:
LPS-171930 Remove Optional usages in ExtendedEntity

julschong committed:
LPS-171930 Remove Optional usages in LiferayMethodDataFetchingProcessor

julschong committed:
LPS-171930 Remove Optional usages in BaseMessageBodyReader

julschong committed:
LPS-171930 Remove Optional usages in BaseMessageBodyWriter

julschong committed:
LPS-171930 Remove Optional usages in GraphQLServletExtender

JanisZhang committed:
LPS-172109 Simplify, getOpenGraphImageFileEntryId should never return null

JanisZhang committed:
LPS-172109 Simplify, getAssetEntryType has default value -> "", should never return null

slnn committed:
LPS-172264 Remove Optional usage in BlogsEntryAnalyticsReportsInfoItem's method body

JanisZhang committed:
LPS-172109 Simplify, SegmentsExperienceId should never return null

slnn committed:
LPS-172264 Remove Optional usage in AnalyticsReportsProductNavigationControlMenuEntry's method body

slnn committed:
LPS-172264 SF, as used

slnn committed:
LPS-172264 Remove Optional usage in TrafficChannel's method body

slnn committed:
LPS-172264 Remove Optional usage in AnalyticsReportsPortlet's method body

slnn committed:
LPS-172264 Remove Optional usage in GetDataMVCResourceCommand's method body

slnn committed:
LPS-172264 Simplify, the list which set by analyticsReportsInfoItem.getActions() will never be a null value

slnn committed:
LPS-172264 Remove Optional usage in AnalyticsReportsLayoutDisplayPageInfoDisplayRequestAttributesContributor's method body

jackelyneaquino committed:
LPS-172130 Deleting before add Site Navegation Menu

felipelorenz committed:
LPS-162052 LPS-165715 portal-search-web: Modified - Replace customRangeModifiedFacetTermDisplayContext with customRangeBucketDisplayContext

https://issues.liferay.com/browse/LPS-165715

Commit:
LPS-170965 Fix permissions

Commit:
LPS-170965 Rename all relationships to use abbreviations

Commit:
LPS-170965 SF

JanisZhang committed:
LPS-171453 Simplify, use TransformUtil instead.

ces-locpham committed:
LPS-172260 Add exception mapper class for DuplicateObjectRelationshipException

slnn committed:
LPS-172264 Remove Optional usage in LayoutAnalyticsReportsInfoItem's method body

slnn committed:
LPS-172264 Remove Optional usage in LayoutDisplayPageObjectProviderAnalyticsReportsInfoItemImpl's method body

thaiscabrals committed:
LPS-163700 Automate FormsSubmissions#CanSeePartialResultsButtonWithLimitsOneSubmissionPerUser

carolmariaabb committed:
LPS-169524 Remove feature flag for LPS-166918 - As a Power User, I want to add an action to be triggered manually through the API

carolmariaabb committed:
LPS-169525 Remove feature flag for LPS-148804 - As a Power User, I want to add an action to be triggered manually by the user

p2kmgcl committed:
LPS-171829 Allow passing a formGroupClassName to AssetTagsSelector

carolmariaabb committed:
LPS-169524 Fix integration test

carolmariaabb committed:
LPS-169524 Fix Poshi Tests

danielCamara02 committed:
LPS-172227 Add new methods in Blueprints path

danielCamara02 committed:
LPS-172227 Fix PaginateBlueprintsAdmin and PaginateBlueprintsPreview and PaginateElementsAdmin tests

tinatian committed:
LPS-171930 SF

Commit:
LPS-172428 Edited Dashboard page for better resposiviness

Commit:
LPS-172075 Refactor in Product Performance Chart

Commit:
LPS-171637 Add Macro viewDisabledButton

Commit:
LPS-171637 Add PRMMDFRequest#CannotContinueWithoutFillRequiredField Test

tinatian committed:
LPS-171930 SF

JanisZhang committed:
LPS-171453 Simplify, remove the constructor without _groupId and make _groupId of type long instead of Long. And remove null check in ContentDashboardItemSubtypeItemSelectorView when getting groupId.

carolmariaabb committed:
LPS-171054 buildService

carolmariaabb committed:
LPS-171054 baseline

p2kmgcl committed:
LPS-171829 Remove bottom margin

p2kmgcl committed:
LPS-171829 Disable input in edit mode

victorg1991 committed:
LPS-172472 Set the window mode to undefined

Otherwise a container class is added to the content div in https://github.com/liferay/liferay-portal/blob/a1a7f4292fc768afec9e7801024d645378140236/modules/apps/frontend-theme/frontend-theme-classic/src/templates/init_custom.ftl#L8 which we need due to LPS-112204. We could've used normal mode but this is changed to maximized in https://github.com/liferay/liferay-portal/blob/4e062b70cd9e8a85a568246bf752b1643821e379/portal-impl/src/com/liferay/portlet/internal/PortletContainerImpl.java#L387 so the easiest option here was used a different window mode that didn't have any other side effect

Commit:
LPS-172428 Refactored React Fragments Container Component Footer

Commit:
LPS-172428 Refactored React Fragments Donut Chart Component

Commit:
LPS-172428 Edited loading Margin

felipelorenz committed:
LPS-162052 LPS-165715 portal-search-web: Modified - Deprecate the use of customRangeModifiedFacetTermDisplayContext to avoid break changes

https://issues.liferay.com/browse/LPS-165715

dantewang committed:
LPS-172109 SF

dantewang committed:
LPS-171453 This null check is necessary

dantewang committed:
LPS-171453 SF

dantewang committed:
LPS-172264 Simplify, User.getFullName() doesn't seem to return null

dantewang committed:
LPS-172264 SF, use JSONUtil.put(String, UnsafeSupplier)

dantewang committed:
LPS-172264 SF, redundant cast

dantewang committed:
LPS-172264 SF, avoid iterate 2 times

Hanlf committed:
LPS-169410 Split tests into different files

p2kmgcl committed:
LPS-172545 Remove unused

riccardo-ferrari committed:
LRAC-12719 analytics-settings-rest-api Update Contact fields

riccardo-ferrari committed:
LRAC-12719 analytics-settings-rest-impl Custom Fields export for Accounts and People

riccardo-ferrari committed:
LRAC-12719 analytics-settings-rest-api Remove unused fields

veroglez committed:
LPS-172557 Add empty text when there is no previously calculated value

Brunno0liveira committed:
COMMERCE-10559 Multiple quantity can be added to cart & Multiple quantity cannot be added to cart Test

biellopes00 committed:
LPS-172570 Fix activities save when draft is selected

riccardo-ferrari committed:
LRAC-12739 analytics-settings-rest-impl Send connection data using form content type

lfbesada committed:
LPS-172580 Use the proper CSS class

tinatian committed:
COMMERCE-10591 Add null checking

luisacmn committed:
LPS-172565 Restricted input fields to numbers

brianchandotcom committed:
LPS-171930 SF

brianchandotcom committed:
LRQA-78789 Sort

Kvale1733 committed:
LRQA-78272 Set test.assert.warning.exceptions to false in ViewServletContext test

felipelorenz committed:
LPS-162052 LPS-165715 portal-search-web: Modified - Sort

https://issues.liferay.com/browse/LPS-165715

brianchandotcom committed:
LRAC-12739 SF

brianchandotcom committed:
LRAC-12719 Faster

brianchandotcom committed:
LRAC-12719 Use transform and transformToArray

brianchandotcom committed:
LPS-171960 Wordsmith

brianchandotcom committed:
LPS-171960 Regen

brianchandotcom committed:
LPS-171052 we should not need to go through aop again since the method
you're calling does not have any indexing annotations

pedro-oliveira446 committed:
LPS-126754 manually setting the tab style

Commit:
LPS-172428 Adding some PR Corrections

shuyangzhou committed:
LPS-172584 Fix broken ObjectResourceBundle parent delegation

kenjiheigel committed:
LRCI-3393 Update relevant to use 'functional-tomcat90-mysql57-chrome100-jdk8'

brianchandotcom committed:
LPS-167775 SF

michaelhashimoto committed:
LRCI-3397 Update com.liferay.jenkins.results.parser to 1.0.1162

brianchandotcom committed:
LPS-171874 SF

kenjiheigel committed:
LRCI-3393 Update stable suite to use *chrome100 batch

brianchandotcom committed:
LRCI-3397 prep next

brianchandotcom committed:
LRCI-3397 prep next

brianchandotcom committed:
LPS-169503 SF

brianchandotcom committed:
LPS-171382 One thing at a time

brianchandotcom committed:
LPS-167763 Avoid stream

oliv-yu committed:
LPS-172173 portal-search-web: Add isSearchExperiencesSupported to SearchBarPortletDisplayContext

oliv-yu committed:
LPS-172173 portal-search-web: Hide blueprints suggestions based on isSearchExperiencesSupported

oliv-yu committed:
LPS-172173 portal-search-web: Update util name initialize_clipboard for consistency

kenjiheigel committed:
POSHI-245 Partial revert of 68edbeda14726064ecf5756d5c21f52269eb3aa6

JanisZhang committed:
LPS-171453 Apply usages

JanisZhang committed:
LPS-171453 Sem Ver

Commit:
LPS-172450 Add macro to go to utility page ellipsis menu item

Commit:
LPS-172450 Add case to mark 404 error utility page as default

brianchandotcom committed:
Revert "LPS-165581 Modified alignmentPosition property"

This reverts commit 6963c22bed626b3b0450b4bbb83b0bbbf09637b1.

brianchandotcom committed:
LPS-172584 SF

brianchandotcom committed:
LPS-171453 Rename

brianchandotcom committed:
LPS-171453 Sort

brianchandotcom committed:
LPS-171453 Consistent naming

brianchandotcom committed:
LPS-172264 Buy space

brianchandotcom committed:
LPS-172109 Inline

brianchandotcom committed:
LPS-166921 SF

Commit:
LPS-172450 Add case to apply 404 error utility page in staging and live site

Commit:
LPS-172450 Add case to change default 404 error utility page

brianchandotcom committed:
Prepare for U60

liferay-translation committed:
LPS-77699 Update Translations

carolmariaabb committed:
LPS-169524 Add object action label and fix object action name

Commit:
COMMERCE-10604 Make ERC updatable through SKU Patch

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

victorcdp commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/30e6ce42fe80240841eb075d9c66b7f39f8b0511">30e6ce42fe80240841eb075d9c66b7f39f8b0511</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/victorcdp/liferay-portal/tree/COMMERCE-10604">COMMERCE-10604</a><br/>Branch GIT ID: <a href="https://github.com/victorcdp/liferay-portal/commit/8dfd462b9d3f8ee810b7af326653b05193057811">8dfd462b9d3f8ee810b7af326653b05193057811</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-26.liferay.com/job/test-portal-source-format/4471/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-source-format/builds/4471/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-26.liferay.com/job/test-portal-source-format/4471/">test-portal-source-format#4471</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-source-format/builds/4471/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/brianikim/liferay-portal/pull/39">brianikim#39</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2159541627">[master] ci:test:sf - victorcdp &gt; brianikim - PR#39 - 2023-01-12[10:13:08]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-26.liferay.com/job/publish-testray-report/16783/">publish-testray-report#16783</a><br/></div></div>

brianikim commented on modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/SkuResourceImpl.java:
```diff
@@ -344,7 +344,7 @@ private Sku _updateSKU(CPInstance cpInstance, Sku sku) throws Exception {
 
 		cpInstance = _cpInstanceService.updateCPInstance(
-			cpInstance.getExternalReferenceCode(), cpInstance.getCPInstanceId(),
+			sku.getExternalReferenceCode(), cpInstance.getCPInstanceId(),
```
@victorcdp , Please resend me a PR to account for the following scenarios.

1. if ERC is provided (with value) the ERC of the cpInstance is updated with the new value
2. if ERC is not provided in the JSON the ERC of the cpInstance is not updated.
3. if ERC is provided as an empty string, the ERC of the cpInstance is not updated.
